gnome-image-installer/pages/install/gis-install-page.c

```diff
@@ -85,8 +85,8 @@ gis_install_page_prepare_read (GisPage *page, GError **error)
   basename = g_file_get_basename(priv->image);
   if (basename == NULL)
     {
-      *error = g_error_new(GIS_INSTALL_ERROR, 0, _("Image verification error: g_file_get_basename(\"%s\") failed"), priv->image);
-      return FALSE;
+      *error = g_error_new (GIS_INSTALL_ERROR, 0, _("Internal error"));
```

Comment:
Reducing the verbosity of the errors is probably going to cause much harder debugging in user-submitted error reports so I'd leave the originals as-is.